internal/usage/usage.go

```diff
@@ -34,14 +34,39 @@ const (
 	FirstName     = "User’s first name."
 	LastName      = "User’s last name."
 	WhitelistIps  = "IP addresses to add to the new user’s whitelist."
-	WhitelistType = `Type of whitelist entry.
-Valid values: cidrBlock|ipAddress`
-	Service = `Type of MongoDB service.
-Valid values: cloud|cloud-manager|ops-manager`
-	Provider = `Name of your cloud service provider.
-Valid values: AWS|AZURE|GCP.`
+	WhitelistType = `Type of whitelist entry. Valid values: cidrBlock|ipAddress`
+	Service       = `Type of MongoDB service. Valid values: cloud|cloud-manager|ops-manager`
+	Provider      = `Name of your cloud service provider.
+										Valid values: AWS|AZURE|GCP.`
```

Comment:
[q] what's with the spacing here? unintended?

Reply:
@andreaangiolillo all these changes to spacing here breaks some help commands
<img width="1579" alt="Screenshot 2020-02-24 at 11 07 11" src="https://user-images.githubusercontent.com/461027/75147694-d2b0cd80-56f5-11ea-9c13-c2ad45beb99d.png">